enterprise/cli/provisionerdaemons.go

```diff
@@ -225,7 +225,7 @@ func (r *RootCmd) provisionerDaemonStart() *clibase.Cmd {
 				cliui.Errorf(inv.Stderr, "Unexpected error, shutting down server: %s\n", exitErr)
 			}
 
-			err = srv.Shutdown(ctx)
+			err = srv.Shutdown(ctx, false)
```

Comment:
Here we're changing the default behavior for interrupt, and since we split interrupt and stop, we're also not handling `SIGTERM` here anymore.

Reply:
Fixed